src/style.css

```diff
@@ -1,5 +1,7 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;600;700&display=swap');
```

Comment:
Please don't fetch the font directly from google but embed it here in the repo. Or even better: import it from the design system.  we've switched from source sans pro to inter a while ago. At least for web. Would make sense to do the same here and would make the import from ods easy.

Reply:
Done - I am copying only the css file now as the link into the head is inserted automatically.